package.json

```diff
@@ -11,7 +11,8 @@
     "storybook": "turbo run storybook --parallel",
     "storybook:build": "turbo run storybook:build --parallel",
     "teardown": "turbo teardown",
-    "test-start": "turbo test-start"
+    "test-start": "turbo test-start", 
+    "dev": "turbo dev" 
```

Comment:
this cmd alr exists `"dev": "dotenv -- turbo run dev"` on main branch

Reply:
removed